src/utils/errorMessage/groupErrorMessage.ts

```diff
@@ -0,0 +1,19 @@
+export enum SubjectError {
+	empty = "Campo vacío",
```

Comment:
Should this value be different from GeneralError.noEmpty?

Reply:
Yes, because do to how the field is implemented, it cannot pass through the general error